src/index.css

```diff
@@ -25,3 +25,40 @@ code {
     'Courier New',
     monospace;
 }
+
+.table {
+  width: 40%;
+  height: 800%;
+  border-collapse: collapse;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.table th,
+td {
+  width: auto;
+  border: solid 1px;
```

Comment:
- [ ] To achieve the given design let us kindly improve the border, replace with `border: 1px solid white`